src/parser/jobs/ast/modules/ArcanaTracking/ArcanaTracking.tsx

```diff
@@ -84,6 +85,7 @@ export default class ArcanaTracking extends Module {
 	static displayOrder = DISPLAY_ORDER.ARCANA_TRACKING
 
 	@dependency private precastStatus!: PrecastStatus
+	@dependency private cooldowns!: Cooldowns
```

Comment:
This does not seem to be used.  Is it intended to be?

Otherwise the PR still looks fine to me.